Frontend/src/Navbar/MapFilters/Mapfilter.js

```diff
@@ -77,19 +102,19 @@ function MapFilter() {
           <div className='flex flex-col gap-y-3 justify-start items-center'>
             <div className='text-2xl border-b-2 border-black p-2'>Filtry</div>
             <div className='flex flex-col gap-2'>
-              <BaseInput
+              {/* <BaseInput
```

Comment:
To samo tutaj?

Reply:
To w ramach późniejszych aktualizacji kodu gdzie będzie można filtrować po nazwie. Trzeba to przedyskutować czy ma być to właśnie filtrowanie w tym miejscu czy w innym komponencie (SearchBar)